DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">results_annual_average = {
  experiment: result.groupby("time.year").mean(dim="time")
  for experiment, result in results.items()
}
</pre><p> </p><p>Alternative with a dictionary comprehension. At the least, loop over <code>results.items()</code> as above. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>I kept the above structure to keep it simple and highlight the annually averaging process in the code.</p>